lib/purescript-cst/README.md

```diff
@@ -1,12 +1,13 @@
 # purescript-cst
 
-Defines the surface syntax of the PureScript Programming Language.
+Defines the surface and underlying syntax of the PureScript Programming Language.
 
 ## Compiler compatibility
 
-We provide a table to make it a bit easier to map between versions of `purescript` and `purescript-cst`.
+We provide a table to make it a bit easier to map between versions of `purescript`, `purescript-cst`, and `purescript-ast`.
```

Comment:
I think now is a good opportunity to put this table in reverse order, so that the latest versions (which people are more likely to care about) are at the top. Also, I think it would be nice to have a separate table following the merge of ast and cst, so that there isn't an extra column. Maybe something like this?

---

We provide a table to make it a bit easier to map between versions of `purescript` and `purescript-cst`.

| `purescript` | `purescript-cst` |
| --- | --- |
| 0.14.2 | 0.2.0.0 |

Before v0.14.2, there was a third package, `purescript-ast`. In v0.14.2, `purescript-ast` was merged into `purescript-cst`.

| `purescript` | `purescript-cst` | `purescript-ast` |
| --- | --- | --- |
| 0.14.1 | 0.1.1.0 | 0.1.1.0 |
| 0.14.0 | 0.1.0.0 | 0.1.0.0 |

Reply:
I like this idea!